src/components/UserDashboard/index.js

```diff
@@ -25,11 +25,12 @@ import {
   userDashboardContent,
 } from './UserDashboard.module.css';
 import { Col, Row } from 'antd';
+import ParticipantSets from './ParticipantSets';
 
 const Container = ({ className = '', children }) => (
   // This is to cancel out the negative margin set by react-grid-system
   <div style={{ marginLeft: '15px', marginRight: '15px' }}>
```

Comment:
if possible, css vs style (even if it s not your code)

Reply:
i think there is task specific to changing the dashboard. i think is a better opportunity to change all this.